### PR TITLE
Tab SSO NODE JS to support authentication in browser. 

### PR DESCRIPTION
**Pull request by HunaidHanfee-MSFT**

### CODE REVIEW

File: samples/tab-sso/nodejs/README.md [hunk 1]
```diff
@@ -82,6 +90,7 @@ Your tab needs to run as a registered Azure AD application in order to obtain an
 2. Update your `config/default.json` file
     * Replace the `tab.id` property with you Azure AD application ID
     * Replace the `tab.password` property with the "client secret" you were assigned in step #2
+    * Replace the `tab.applicationIdUri` propert with the Application ID URI we get in step #1.1 above. It will look like this - `api://contoso.ngrok.io/{appID}`
```
Comment (ChetanSharma-msft) [inline samples/tab-sso/nodejs/README.md:93]: propert => property

File: samples/tab-sso/nodejs/appPackage/manifest.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+   "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+   "manifestVersion": "1.11",
```
Comment (ChetanSharma-msft) [inline samples/tab-sso/nodejs/appPackage/manifest.json:3]: Should we use 1.12 ?

File: samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js [hunk 3]
```diff
@@ -88,41 +89,47 @@
...
+                .then((profile) => {
+                    return useServerSideToken(profile);
+                })
+                .catch((error) => {
+                    if (error === "invalid_grant") {
+                        display(`Error: ${error} - user or admin consent required`);
+                        // Display in-line button so user can consent
+                        let button = display("Consent", "button");
+                        button.onclick = (() => {
+                            requestConsent()
```
Comment (ChetanSharma-msft) [inline samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js:107]: Add proper spacing and line break before if, let etc.

File: samples/tab-sso/nodejs/src/server/tabs.js [hunk 4]
```diff
@@ -13,7 +13,9 @@ module.exports.setup = function (app) {
 
   // Setup home page
   app.get('/', function (req, res) {
-    res.render('hello');
+    var clientId = config.get("tab.appId");
+    var applicationIdUri = config.get("tab.applicationIdUri");
+    res.render('hello', { clientId: clientId, applicationIdUri: applicationIdUri });
```
Comment (ChetanSharma-msft) [inline samples/tab-sso/nodejs/src/server/tabs.js:18]: Rename res to some meaningful name

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Reviewed and provided my comments

File: samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js (hunk 3 above)
Comment (HunaidHanfee-MSFT) [reply samples/tab-sso/nodejs/src/client/scripts/ssoDemo.js:112]: Added

File: samples/tab-sso/nodejs/appPackage/manifest.json (hunk 2 above)
Comment (HunaidHanfee-MSFT) [reply samples/tab-sso/nodejs/appPackage/manifest.json]: Yes, Updated

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

File: samples/tab-sso/nodejs/README.md [hunk 5]
```diff
@@ -132,6 +141,16 @@ Thhe app should start running on port 3333 or the port you configured
 
 ![SSO from a mobile device](./doc/images/AAD-SSO-Tab-6-Mobile.png)
 
+* You can test the SSO in browser as well.
+
+To test SSO in Browser, open your app base URL or Ngrok URI in your browser. 
+
+The URI shoud be like `https://contoso.ngrok.io/`
+
+![Browser SSO](./doc/images/Browser-SSO.png)
```
Comment (Wajeed-msft) [inline samples/tab-sso/nodejs/README.md:150]: Could you please change this to "Login Outside Microsoft Teams"? Also, rename the image.

**Review by Wajeed-msft — APPROVED** [review]

File: samples/tab-sso/nodejs/README.md (hunk 5 above)
Comment (HunaidHanfee-MSFT) [reply samples/tab-sso/nodejs/README.md:150]: Done changed

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**